utils/data-processing-family-task-submitter/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>com.github.cafdataprocessing</groupId>
         <artifactId>processing-utilities-aggregator</artifactId>
-        <version>1.1.0-SNAPSHOT</version>
+        <version>1.2.0-SNAPSHOT</version>
     </parent>
```

Comment:
This probably should be in separate commit as it's not related to this work. Not too worried about it though.